src/lib.rs

```diff
@@ -24,3 +24,6 @@ pub mod common;
 pub mod delay;
 pub mod i2c;
 pub mod spi;
+
+/// Mock InputPin and OutputPin implementations
```

Comment:
I'd put mod documentation inside the module (with `//! comments`). Then you have a description of what the file does when you open it :slightly_smiling_face:

Also, another paragraph with a quick explanation on how to use this mock and a small example would be nice!